cfgmgr/intfmgr.cpp

```diff
@@ -288,22 +303,158 @@ void IntfMgr::addHostSubIntf(const string&intf, const string &subIntf, const str
     EXEC_WITH_ERROR_THROW(cmd.str(), res);
 }
 
-void IntfMgr::setHostSubIntfMtu(const string &subIntf, const string &mtu)
+
+std::string IntfMgr::getIntfAdminStatus(const string &alias)
+{
+    Table *portTable;
+    if (!alias.compare(0, strlen("Eth"), "Eth"))
+    {
+        portTable = &m_statePortTable;
+    }
+    else if (!alias.compare(0, strlen("Po"), "Po"))
+    {
+        portTable = &m_appLagTable;
+    }
+    else
+    {
+        return "down";
+    }
+
+    string admin = "down";
```

Comment:
please have this as the beginning of the function and return admin variable instead of repeating "down" key word

Reply:
Fixed as part of next commit